packages/node_modules/@webex/internal-plugin-devices/test/unit/spec/features/feature-model.js

```diff
@@ -3,137 +3,139 @@ import {FeatureModel} from '@webex/internal-plugin-devices';
 
 import dto from '../wdm-dto';
 
-describe('feature-model', () => {
-  let featureLM;
-  let featureNLM;
-  let featureModel;
-
-  beforeEach('create feature model', () => {
-    [featureLM, featureNLM] = dto.features.developer;
-  });
-
-  describe('#constructor()', () => {
-    describe('when the feature includes a \'lastModified\' property', () => {
-      beforeEach('generate the feature model', () => {
-        featureModel = new FeatureModel(featureLM);
-      });
-
-      it('should assign value attributes to properties', () => {
-        assert.equal(featureLM.key, featureModel.key);
-        assert.equal(featureLM.mutable, featureModel.mutable);
-        assert.equal(featureLM.type, featureModel.type);
-        assert.equal(featureLM.val, featureModel.val);
-        assert.equal(featureLM.value, featureModel.value);
-      });
-
-      it('should assign the \'lastModified\' value as a \'Date\'', () => {
-        assert.instanceOf(featureModel.lastModified, Date);
-
-        assert.equal(
-          featureModel.lastModified.toISOString(),
-          featureLM.lastModified
-        );
-      });
+describe('plugin-devices', () => {
```

Comment:
This file can mostly be ignored as I fixed an indention mismatch due to a missing `describe()`.